### ISSUE TITLE
Small difference between code and paper

### ISSUE THREAD

**Issue by os920090so** [rosinality/swapping-autoencoder-pytorch#16]
Hi.
Thanks for sharing the excellent code!
Compare with [paper ](https://arxiv.org/abs/2007.00653) Appendix B, I find 2 differences

1. Final layer of the encoder is ConvLayer. But it's Dense layer in paper.

https://github.com/rosinality/swapping-autoencoder-pytorch/blob/2c50ad602635423ebb87b218f052792c08e118b0/model.py#L252-L257
![image](https://user-images.githubusercontent.com/25976586/100497190-aa7a6180-3194-11eb-958a-04662c878c8a.png)

2.Discriminator use L1 loss after softplus activation, binary cross-entropy in paper.
![image](https://user-images.githubusercontent.com/25976586/100497372-14473b00-3196-11eb-97cf-4ce8012e301f.png)

**Comment by rosinality** [comment]
1. As global average pooling is used, conv will be same with dense layer.
2. Using softplus is equal to binary cross entropy gan loss.

**Comment by os920090so** [comment]
I get it. Thanks for the explanation!
